proto/lavanet/lava/pairing/genesis.proto

```diff
@@ -21,11 +19,34 @@ message BadgeUsedCu {
 // GenesisState defines the pairing module's genesis state.
 message GenesisState {
   Params params = 1 [(gogoproto.nullable) = false];
-  repeated UniquePaymentStorageClientProvider uniquePaymentStorageClientProviderList = 2 [(gogoproto.nullable) = false];
-  repeated ProviderPaymentStorage providerPaymentStorageList = 3 [(gogoproto.nullable) = false];
-  repeated EpochPayments epochPaymentsList = 4 [(gogoproto.nullable) = false];
+  reserved 2;
+  reserved 3;
+  reserved 4;
   repeated BadgeUsedCu badgeUsedCuList = 5 [(gogoproto.nullable) = false];
   lavanet.lava.timerstore.GenesisState badgesTS = 6 [(gogoproto.nullable) = false];
   lavanet.lava.fixationstore.GenesisState providerQosFS = 7 [(gogoproto.nullable) = false];
+  repeated UniqueEpochSessionGenesis unique_epoch_sessions = 8 [(gogoproto.nullable) = false];
+  repeated ProviderEpochCuGenesis provider_epoch_cus = 9 [(gogoproto.nullable) = false];
+  repeated ProviderConsumerEpochCuGenesis provider_consumer_epoch_cus = 10 [(gogoproto.nullable) = false];
   // this line is used by starport scaffolding # genesis/proto/state
 }
+
+message UniqueEpochSessionGenesis {
+  uint64 epoch = 1;
+  string unique_epoch_session = 2;
```

Comment:
lets save the decoded key like the others

Reply:
done. see https://github.com/lavanet/lava/commit/35e283c21cc5ab8236c48bfac72624a1821d4898